doxia-core/src/main/java/org/apache/maven/doxia/index/IndexEntry.java

```diff
@@ -304,7 +310,7 @@ public String toString(int depth) {
             message.append(", title: ").append(title);
         }
 
-        message.append(EOL);
+        message.append(System.lineSeparator());
```

Comment:
Attention, this is a semantical change. All others use the idiom above and there are tests which change the separator with try/finally. This one is caches upon first call. I wouldn't change it.

Reply:
I reverted that change now @michael-o